src/array/cholesky.jl

```diff
@@ -0,0 +1,82 @@
+is_cross_symmetric(A1, A2) = A1 == A2'
+function LinearAlgebra.issymmetric(A::DArray{T,2}) where T
+    Ac = A.chunks
+    if size(Ac, 1) != size(Ac, 2)
+        return false
+    end
+
+    to_check = [Dagger.@spawn issymmetric(Ac[i, i]) for i in 1:size(Ac, 1)]
+    for i in 2:(size(Ac, 1)-1)
+        j_pre_diag = i - 1
+        for j in 1:j_pre_diag
+            push!(to_check, Dagger.@spawn is_cross_symmetric(Ac[i, j], Ac[j, i]))
+        end
+    end
+
+    return all(fetch, to_check)
+end
+function LinearAlgebra.ishermitian(A::DArray{T,2}) where T
+    Ac = A.chunks
+    if size(Ac, 1) != size(Ac, 2)
+        return false
+    end
+
+    to_check = [Dagger.@spawn ishermitian(Ac[i, i]) for i in 1:size(Ac, 1)]
+    for i in 2:(size(Ac, 1)-1)
+        j_pre_diag = i - 1
+        for j in 1:j_pre_diag
+            push!(to_check, Dagger.@spawn is_cross_symmetric(Ac[i, j], Ac[j, i]))
+        end
+    end
+
+    return all(fetch, to_check)
+end
+
+LinearAlgebra.cholcopy(A::DArray{T,2}) where T = copy(A)
+function LinearAlgebra.cholesky!(A::DArray{T,2}, ::NoPivot; check::Bool=true) where T
```

Comment:
I think we have to be explicit when using `check` word, because it could refer to correctness of factorization or of the matrix specific properties

Reply:
What would be a better word to use than `check`? That's what the `cholesky!(::Array)` implementation in LinearAlgebra uses.

Reply:
No longer used in the latest commits